service/github/github.go

```diff
@@ -206,6 +211,16 @@ func (g *PullRequest) postAsReviewComment(ctx context.Context) error {
 		}
 	}
 
+	for _, c := range g.outdatedComments {
+		if ok := g.prCommentWithReply[c.GetID()]; ok {
+			// Do not remove comment with replies.
+			continue
```

Comment:
I think it would be easier to understand that reviewdog create a reply like `This review comment has been resolved.` if there is a outdated comment with replies.

Reply:
Thanks, I agree with you to some extent, but if we'll create a reply comment, then we need to check whether we already have the reviewdog reply or not to avoid posting duplicated replies. 

It  needs extra logic and maitainance cost. I prefer keeping it as-is, at least for now, unless we have a simple way to handle this problem.